recipe/meta.yaml

```diff
@@ -43,8 +42,8 @@ test:
     - pip check
     - tqdm --help
     - tqdm -v
-    - pytest -k "not tests_perf"
-
+    - pytest -k "not tests_perf and not test_pipes" #[win]
```

Comment:
test_pipes uses ls which is likely not available on prefect win builders.
Perhaps it was available on concourse win builders in the past.
Either way, this is fine.

Reply:
Thanks! However, without this restriction, the win-64 build fails as a result of test_pipes not being compatible with this particular platform. In the code test_pipes, it utilizes "ls" which can not be used for win-64.